City-of-Helsinki/drupal-hdbt
# UHF-5662: Added styles for hero with search

# Hero with search [UHF-5662](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5662)

Added new design for hero

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme
    * `composer require drupal/hdbt:dev-UHF-5662-hero-with-search`
* Run `make drush-cr`

## Designers review

One of the checkboxes below needs to be checked (`[x]`)


## How to test
* See this [PR ](https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/40) for instructions
